lib/scsi_defs.h

```diff
@@ -80,5 +85,23 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  //SCSI Reference Manual Table 202 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  typedef struct TestUnitReadyCommand {
+    OpCode opCode = OpCode::kTestUnitReady;
+    uint32_t reserved;
+    ControlByte controlByte;
+    uint16_t padding;   // Necessary for bit_cast
```

Comment:
So the `TestUnitReadyCommand` as per the specs is 48 bits (6 bytes). But when I make a `bitset<48>`, the size of that is actually 8 bytes instead of 6 bytes. So to allow for `bit_cast` which requires the `sizeof()` to be the same, I padded my struct with 2 bytes so it can be 8 bytes.

Reply:
What's the benefit of caring about `bitset` here? Ultimately we'll be wanting to convert to/from buffers of raw bytes containing the specified ABI, so we should try to respect that ABI exactly. In other words, if the spec says a structure should be 6 bytes, the definition should probably be 6 bytes.

Reply:
I'm concerned that changing the size will lead to problems when we're looking at actual scsi command data - those aren't going to include padding bytes for us.  If bitset isn't working, we might have to fall back on more old fashioned methods such as C-style byte arrays.

Reply:
`std::array<uint8_t, 6>` would also be a nice robust approach.